webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -545,7 +560,6 @@ class VolumeTracingController @Inject()(
           _ <- bool2Fox(request.body.length == 1) ?~> "Editable mapping update group must contain exactly one update group"
           updateGroup <- request.body.headOption.toFox
           _ <- bool2Fox(updateGroup.version == currentVersion + 1) ?~> "version mismatch"
-          _ <- bool2Fox(updateGroup.actions.length == 1) ?~> "Editable mapping update group must contain exactly one update action"
```

Comment:
Is there anything else we should be aware of? E.g. are the actions applied in the order intended? Can the frontend now also send multiple actions in one group?

Reply:
I’d say this should work as is. For this particular update group, the order should actually not matter, as it deletes all the edges of the cut.
Multiple actions in one group is exactly what is now possible. It still has to be one group.